---
title: "Obj B Protocol"
author: "Jacob M. Green"
date: "9/8/2020"
output: html_document
---

```{r setup, include=FALSE}
knitr::opts_chunk$set(echo = TRUE)
```

# EecSeq Lab Protocol

The Expressed Exome Capture Sequencing protocol is designed to create exome capture probes directly from RNA.  The probes are then used from hybrid capture of exome DNA sequences, allowing for genotyping of alleles at expressed genes.

#### This protocol is still in the process of being optimized.  Please contact Jon Puritz (jpuritz@uri.edu) for more information before starting!

# Outline

[RNA Prep](#rna-prep)
* [RNA Extraction](#extract-rna-from-individuals-to-be-used-for-probes)
* [Quantify](#quantify-all-rna-samples)
* [Visualize](#visualize-rna-on-Tape-Station)
* [mRNA Library Prep](#stranded-mrna-seq-library-prep)
	* [Anneal adapters](#anneal-rna-adapters)
	* [mRNA capture](#mrna-capture)
	* [mRNA Elution, Fragmentation, and Priming](#mrna-elution-fragmentation-and-priming)
	* [1st Strand Synthesis](#1st-strand-synthesis)
	* [2nd Strand Synthesis](#2nd-strand-synthesis-and-marking)
	* [A-tailing](#a-tailing)
	* [Adapter Ligation](#adapter-ligation)
	* [Library Enrichment](#library-amplificiation)
* [Quant](#quant-libraries)
* [DSN Normalization](#dsn-normalization)
	* [Enrichment](#pcr-enrichment)
* [Quant Libraries](#quant-libraries)
* [Split libraries](#split-finished-cdna-library-for-each-sample-into-two-vials)

[Probe Synthesis](#probe-synthesis)
* [Remove sequencing adapters](#remove-adapters-from-cdna)
* [Biotin Labeling](#biotin-labeling)

[Genomic Library Prep](#preparation-of-whole-genome-libraries)
* [Anneal Adapters](#anneal-adapters)
* [End Repair](#end-repair)
* [Adapter ligation](#adapter-ligation)
* [Quantification](#quant-samples)
* [Amplification](#library-amplification)

[Hybridization and Capture](#hybridization-and-capture)
* [Blocking Oligos](#blocking-oligos)
* [Hybridization](#hybridization)
	* [Preparation of DynaBeads](#preparation-of-dynabeads)
	* [Washes](#washes)
* [Library Re-amplification](#library-re-amplification)
* [Quant Final Libraries](#quant-samples)
* [Verify](#verify)



---
Before you begin this protocol, you should order the custom oligos required for mRNA library prep and genomic DNA library prep and anneal them. See [Anneal RNA adapters](#anneal-rna-adapters) and [Anneal DNA Adapters](#anneal-adapters)


## RNA Prep

### Extract RNA from individuals to be used for probes
*Refer to manual during procedure (steps below are for notes and comments)*
### Using slightly modified Zymo Duet DNA/RNA extraction protocol [LINK](https://www.zymoresearch.com/media/amasty/amfile/attach/_D7003_ZR-Duet_DNA-RNA_MiniPrep_Plus_ver_1.0.1.pdf) which will extract both DNA and RNA at the same time (Below are summary steps)

#### Reagents and supplies
* RNase-free Water
* 100% ethanol, ACS grade or better
* 10mM Tris HCl pH 8.0 made with RNase-free water

#### Equipment
* Rocking oven that can be set to 55°C
* RNase away and a designated RNase free space
* Tabletop and larger centrifuges for 1.5mL and 50mL tubes capable of 12,000 x g

**Notes before starting**
* Wipe down benchtop with RNase away and have the spray bottle and kimwipes on-hand to use frequently

#### Procedure
* Set up 8 50mL conical tubes, labeled A, B, C, D, E, F, G, H. In each add 1mL of DNA/RNA shield
* Wipe down scalpel handle with RNase away and a kimwipe (do this before cutting each filter) and attach new blade and keep inside foil until ready to use
* Take a whirl pak out of the -80 freezer and unwrap the filter. Briefly look at the filter to determine roughly if there is some, few, or none present of visible larvae. Save the whirl pak
* Fit the filter over the openning to the first conical tube. Using the scalpel slice a cross in the center of the filter, then following the edge of the filter slice out the filter so that 4 triangles of filter fall into the tube and into the DNA/RNA sheild.
* Record the information from the whirl pak in notebook correseponding to the labeled tube
* Repeat above steps for each filter
* Add 100μL PK digestion buffer to each conical tube
* Add 50μL Proteinase K to each conical tube and votex. Make sure the filter triangles stay submerged
* Place in incubator genie for 3 hours at 55°C rocking at 35 speed. Vortex every 30 minutes
* Prepare tubes for next steps. Set up sixteen 5mL tubes labeled with letters and 8 for DNA, and the other 8 for RNA, 8 1.5mL tubes for DNA, and 8 each of collection tubes and spin columns. Green are for RNA and yellow are for DNA
* Once the digestion has completed, centrifuge the 50mL conical tubes for 2 minutes at maximum RCF in a large centrifuge
* Transfer all liquid (~1150μL) from 50mL tubes to the corresponding 1.5mL tubes labeled DNA. Try not to transfer any small filter bits, _dab at the filter on the sides of the tube to push out liquid_.
* Centrifuge 1.5mL tubes in benchtop centrifuge for 2 minutes at 20,000 RCF
* Remove supernatent from each tube and put in corresponding 5mL tube. Save the tube with the pellet for imaging
* **In the hood** Add 1mL (roughly equal parts) DNA/RNA lysis buffer to each 5mL tube. Finger flick tubes to mix
* **Rest not in hood**. Transfer 700μL of this liquid to the corresponding **yellow** spin column. Centrifuge at 16,000 RCF for 30 seconds
* **Transfer flow-through to the corresponding 5mL tubes labeled for RNA**
* Repeat the previous 2 steps until all liquid has gone through the columns
* Add 400μL DNA/RNA Prep Buffer to each DNA column and centrifuge at 16,000 RCF for 30 seconds. Discard flow-through
* Add 700μL DNA/RNA Wash Buffer to each DNA column and centrifuge at 16,000 RCF for 30 seconds. Discard flow-through
* Add 400μL DNA/RNA Wash Buffer to each DNA column and centrifuge at 16,000 RCF for **2 minutes**. Discard flow-through
* Transfer columns to new 1.5mL tubes and add 50μL warm (70°C, you can warm in the incubator genie) 10mM Tris HCl pH 8.0 directly to the filter in each column. Incubate at room temp for 5 minutes. Centrifuge at 16,000 RCF for 30 seconds. Keep flow-through in tube
* Repeat previous step. Once done, you have eluted 100μL extracted DNA for each filter. Label tubes approprately and store in 4°C if using within 1 week, if not put in -20°C
* _Return to the 5mL tubes labeled RNA. Alternatively you can perform both column centrifugation at the same time if you keep track of what step you are on for the two extractions_ Add 2mL (~equal volume) of 100% EtOH to each 5mL tube and votex to mix
* Transfer 700μL of this liquid to the corresponding **green** spin column. Centrifuge at 16,000 RCF for 30 seconds
* Discard flow-through
* Repeat previous two steps until all liquid has gone through the columns
* Add 400μL DNA/RNA Wash Buffer to each DNA column and centrifuge at 16,000 RCF for 30 seconds. Discard flow-through
* Make DNase I master mix: [75μL DNA digestion buffer and 5μL DNase] X n (sample #)
* Add 80μL of DNase I master mix directly to the filter of each column. Incubate at room temperature for ~15 minutes
* Add 400μL DNA/RNA Prep Buffer to each DNA column and centrifuge at 16,000 RCF for 30 seconds. Discard flow-through
* Add 700μL DNA/RNA Wash Buffer to each DNA column and centrifuge at 16,000 RCF for 30 seconds. Discard flow-through
* Add 400μL DNA/RNA Wash Buffer to each DNA column and centrifuge at 16,000 RCF for **2 minutes**. Discard flow-through
* Transfer columns to new 1.5mL tubes and add 50μL warm (70°C, you can warm in the incubator genie) RNase-free water directly to the filter in each column. Incubate at room temp for 5 minutes. Centrifuge at 16,000 RCF for 30 seconds. Keep flow-through in tube
* Repeat previous step. Once done, you have eluted 100μL extracted RNA for each filter. Label tubes approprately and keep on ice if moving directly to quant, or aliqiot out ~4μL for Qubit and Tape Station into separtate strip tubes to avoid freeze-thaw of stock, which is kept in -80°C


### Quantify all RNA samples
Results will be used for calibration points during library generation
Refer to manual during procedure (steps below are for notes and comments)

#### Reagents and supplies
* Qubit® RNA HS Assay Kit (ThermoFisher Q32852)
* Microcentrifuge tubes for florescence (Fisher Catalog # 14-222-292)

#### Equipment
* Qubit® 3.0 Flourometer

#### Procedure (Standard HS RNA protocol)
* Set up the required number of 0.5-mL tubes for standards and samples. The Qubit® RNA HS Assay requires 2 standards.
* Label the tube lids.
* Prepare the Qubit® working solution by diluting the Qubit® RNA HS Reagent 1:200 in Qubit® RNA HS Buffer. Use a clean plastic tube each time you prepare Qubit® working solution. **Do not mix the working solution in a glass container.**
* Add 190 μL of Qubit® working solution to each of the tubes used for standards.
* Add 10 μL of each Qubit® standard to the appropriate tube, then mix by vortexing 2–3 seconds. Be careful not to create bubbles.
* Add Qubit® working solution to individual assay tubes so that the final volume in each tube after adding sample is 200 μL.
* Add each sample to the assay tubes containing the correct volume of Qubit® working solution, then mix by vortexing 2–3 seconds. The final volume in each tube should be 200 μL.
* Allow all tubes to incubate at room temperature for 2 minutes.
* On the Home screen of the Qubit® 3.0 Fluorometer, press RNA, then select RNA: High Sensitivity as the assay type. The “Read standards” screen is displayed. Press Read Standards to proceed.
* Insert the tube containing Standard #1 into the sample chamber, close the lid, then press Read standard. When the reading is complete (~3 seconds), remove Standard #1.
* Insert the tube containing Standard #2 into the sample chamber, close the lid, then press Read standard. When the reading is complete, remove Standard #2.
* Press Run samples.
* On the assay screen, select the sample volume and units
* Insert a sample tube into the sample chamber, close the lid, then press Read tube. When the reading is complete (~3 seconds), remove the sample tube.
* Repeat step last step until all samples have been read

## Visualize RNA on Tape Station

### (Agilent Technologies 4200 Tape Station)

* Take out RNA tape, loading buffer, and ladder and allow for 30 min to aquilibrate to room temp
* Thaw sample RNA and ladder on ice
* Using strip tubes and caps from Agilent Technologies, label the first well as the ladder, and the rest for your samples
* Add 5μL RNA sample buffer to each tube you will use, inculding the one for the ladder
* Add 1μL RNA ladder in the first well. Add 1μL RNA sample to the corresponding wells
* Put on tube caps and mix in IKA vortexer for 1 minute
* Centrifuge down tubes
* Put in Thermocycler RNA de-nature program (72°C for 3 minutes then down to 4°C) and put on ice (note, mollusc samples don't really need this)
* Centrifuge down tubes
* Turn on laptop and Tape Station and let it warm up
* Put in tape and load in samples (take off cap) and press start

**Genomic DNA and smaller fragment DNA Tape Station protocols are basically the same, just using the appropriate buffer, ladder, and tape, does not include the de-nature step, and uses 10μL of buffer**

## Stranded mRNA Seq Library Prep

#### Reagents

KAPA Stranded mRNA-Seq Kit (KAPA #KK8420). This kit includes all the enzymes and buffers required for cDNA library preparation from  isolation of poly(A)-tailed RNA. Kits include reagents for RNA fragmentation, 1st strand cDNA synthesis and 2nd strand synthesis/marking, and cDNA library preparation, including A-tailing, ligation and library amplification.

Steps in Library construction:

* mRNA capture using magnetic oligo-dT beads
* Fragmentation using heat and magnesium
* 1st Strand cDNA Synthesis using random priming
* 2nd Strand cDNA Synthesis and marking, which converts the cDNA:RNA hybrid to double-stranded cDNA (dscDNA) and incorporates dUTP in the second cDNA strand
* A-tailing to add dAMP to the 3′-ends of the dscDNA library fragments
* Adapter ligation, where dsDNA adapters with 3′-dTMP overhangs are ligated to A-tailed library insert fragments
	* **NOTE** Here, we insert custom adapters.  See below.
* Library amplification to amplify library fragments carrying appropriate adapter sequences at both ends using high-fidelity, low-bias PCR; the strand marked with dUTP is not amplified.

#### Additional reagents needed:

1. Annealing buffer stock (10X):

| Component| Concentration|
|----------|--------------|
| Tris HCl, pH 8| 100 mM|
| NaCl|500 mM|
| EDTA| 10 mM|

2. 10 mM Tris-HCL (pH 8.0 - 8.5)
#### Equipment

* Magnetic stand and compatible tubes or striptubes
* Thermocycler
* SPRI purification beads (KAPA Pure Beads or AmpureXP)

#### Custom Oligos needed to make adapters:

![alt text](/RNAadapter.png)


|Oligo Name| Sequence|
|----------|---------|
|Universal_SAI1_Adapter|AATGATACGGCGACCACCGAGATCTACACTCTTTCCCTACACGACGCTCTTCCGATCTGTCGACT*T|
|Indexed_Adapter_SAI1_I5|P*AGTCGACAGATCGGAAGAGCACACGTCTGAACTCCAGTCACACAGTGATCTCGTATGCCGTCTTCTGCTTG|
|Indexed_Adapter_SAI1_I8|P*AGTCGACAGATCGGAAGAGCACACGTCTGAACTCCAGTCACACTTGAATCTCGTATGCCGTCTTCTGCTTG|
|Indexed_Adapter_SAI1_I9|P*AGTCGACAGATCGGAAGAGCACACGTCTGAACTCCAGTCACGATCAGATCTCGTATGCCGTCTTCTGCTTG|
|Indexed_Adapter_SAI1_I11|P*AGTCGACAGATCGGAAGAGCACACGTCTGAACTCCAGTCACGGCTACATCTCGTATGCCGTCTTCTGCTTG|

*Note you cannot see the full sequence in GitHub, please look at the raw code for the whole sequence

### Anneal RNA Adapters
Single-stranded oligos need to be annealed with their appropriate partner before ligation.

* To create Adapter SAI1_I5, combine Universal_SAI1_Adapter with Indexed_Adapter_SAI1_I5 in a 1:1 ratio in working strength annealing buffer (final buffer concentration 1x) for a total annealed adapter concentration of 40uM (for example, if purchased oligos are resuspended to an initial concentration of 100uM, use 40uL Universal_SAI1_Adapter, 40ul Indexed_Adapter_SAI1_I5, 10ul 10x annealing buffer and 10ul nuclease-free water). Pair Universal_SAI1_Adapter with Indexed_Adapter_SAI1_I8, Indexed_Adapter_SAI1_I9, Indexed_Adapter_SAI1_I11 in the same fasion.
2.   In a thermocyler, incubate at 97.5°C for 2.5 minutes, and then cool at a rate of not greater than 3°C per minute until the solution reaches a temperature of 21°C. Hold at 4°C.
3.   Prepare final working strength concentrations of annealed adapters from this annealed stock (the appropriate working stock dilution for your experiment can be determined from the chart below). For convenience, it is possible to store the adapters at 4°C while in active use.  

#### Adapter concentration will vary depending on overall RNA yield, see table below:
|Quantity of starting material|Adapter stock concentration|Adapter concentration in ligation reaction|
|----|----|-----|
|100 – 250 ng|140 nM| 10 nM|
|251 – 500 ng|350 nM|25 nM|
|501 – 2000 ng|700 nM|50 nM|
|2001 – 4000 ng|1400 nM|100 nM|

For Puritz and Lotterhos 2017, we used 4000 ng starting RNA, but because of difficulties assessing and quantifying molluscan RNA, we chose to use a 700 nM working stock with a final reaction concentration of 50 nM.

**Procedure:**
* Prepare mastermixes for number of libraries (individual RNA extractions)
  * See tables in manual and tables below for guidelines (**We are using 1/2 reactions from the kit**):

|Component| Total Volume Needed for 4 RXNs (Includes 20% excess)|
|---------|--------------------|
|**1st Strand Synthesis Master Mix:**|--|
|1st Strand Synthesis Buffer|22 μl|
|KAPA Script|2 μl|
|**Total Master Mix Volume**| **24 μl**|
|**Final reaction composition:**|--|
|1st Strand Synthesis Master Mix| 5 μl|
|Fragmented, primed RNA|10 μl|
|**Total Reaction Volume**| **15 μl**|

|Component| Total Volume Needed for 4 RXNs (Includes 10% excess)|
|---------|--------------------|
|**2nd Strand Synthesis and Marking Master Mix:**|--|
|2nd Strand Marking Buffer|62 μl|
|2nd Strand Synthesis Enzyme Mix|4 μl|
|**Total Master Mix Volume**| **66 μl**|
|**Final reaction composition:**|--|
|2nd Strand Synthesis and Marking Master Mix| 15 μl|
|Fragmented, primed RNA|15 μl|
|**Total Reaction Volume**| **30 μl**|

|Component| Total Volume Needed for 4 RXNs (Includes 10% excess)|
|---------|--------------------|
|**A-Tailing Uninterrupted Protocol Master Mix:**|--|
|Water|52.8 μl|
|10X KAPA A-Tailing Buffer|6.6 μl|
|KAPA A-Tailing Enzyme|6.6 μl|
|**Total Master Mix Volume**| **66 μl**|

|Component| Total Volume Needed for 4 RXNs (Includudes 10% excess)|
|---------|--------------------|
|**A-Tailing at Safe Stopping Master Mix:**|--|
|Water| 29.7 μl|
|10X KAPA A-Tailing Buffer| 3.3 μl|
|**Total Master Mix Volume**| 33 μl|
|**Volume to resuspend beads in after 2nd strand syntheis clean up**| 7.5 μl|

|Component| Total Volume Needed for 4 RXNs (Includudes 10% excess)|
|---------|--------------------|
|**A-tailing _After_ Safe Stopping Master Mix**|--|
|Water| 23.1 μl|
|10X KAPA A-Tailing Buffer| 3.3 μl|
|KAPA A-Tailing Enzyme| 6.6 μl|
|**Total Master Mix Volume**| 33 μl|

|Component| Total Volume Needed for 4 RXNs (Includes 10% excess)|
|---------|--------------------|
|**Adapter Ligation Master Mix:**|--|
|Water|35.2 μl|
|5X KAPA Ligation Buffer| 30.8μl|
|KAPA T4 DNA Ligase|11 μl|
|**Total Master Mix Volume**| **77 μl**|
|**Final reaction composition:**|--|
|Beads with A-tailed DNA|15 μl|
|Adapter Ligation Master Mix|17.5 μl|
|Adapter (350 nM – 1400 nM, as appropriate)|2.5 μl|
|**Total Reaction Volume**| **35 μl**|

|Component| Total Volume Needed for 4 RXNs (Includes 10% excess)|
|---------|--------------------|
|**Library Amplification Master Mix:**|--|
|2X KAPA HiFi HotStart ReadyMix|55 μl|
|10X KAPA Library Amplication Primer Mix|11 μl|
|**Total Master Mix Volume**| **66 μl**|
|**Final reaction composition:**|--|
|Adapter-ligated library DNA|10 μl|
|Library Amplification Master Mix|15 μl|
|Balance of water (if required)|5 μl|
|**Total Reaction Volume**| **30 μl**|

### mRNA Capture

* Before mRNA capture beads can be used they must be washed with mRNA Bead Binding Buffer
	* Resuspend beads thoroughly by gentle pipetting or vortexing
	* For each library to be prepared, transfer 26.25 uL of the resuspended mRNA Capture beads into an appropriate tube
		* Up to 48 libraries (1,260 uL) can be washed in a single tube
	* Place the tube on a magnet holder and incubate at room temperature until solution is clear.
	* Discard supernatant and replace with an equal volume of mRNA Bead Binding Buffer.
	* Remove tube from magent and again resuspend the beads.
	* Place the tube on a magnet holder and incubate at room temperature until solution is clear.
	* Discard supernatant and replace with an equal volume of mRNA Bead Binding Buffer.
	* Remove tube from magent and again resuspend the beads.

* Combine the following for each RNA sample to be captured:

|Component|Volume|
|---------|------|
|RNA sample (in RNase-free water)| 25 μl|
|KAPA mRNA Capture Beads| 25 μl|
|**Total Volume**| **50 μl**|

* Mix thoroughly by gently pipetting up and down several times.
* Place the plate/tube in a thermal cycler and carry out the 1st mRNA capture program as follows:

|Step|Temp.|Duration|
|----|-----|--------|
|1st mRNA capture|65 °C|2 min|
|Cool|20 °C|5 min|


* Place the plate/tube containing the mixture of KAPA mRNA Capture Beads and RNA on a magnet and incubate at room temperature until the solution is clear. Remove and discard the supernatant.
* Remove the plate/tube from the magnet and resuspend thoroughly in 100 μl of KAPA mRNA Bead Wash Buffer by pipetting up and down several times.
* Place the plate/tube on the magnet and incubate at room temperature until the solution is clear. Remove and discard the supernatant.
* Resuspend the beads in 25 μl of RNase-free water.
* Place the plate/tube in a thermal cycler and carry out the 2nd mRNA capture program as follows:

|Step|Temp.|Duration|
|----|-----|--------|
|2nd mRNA capture|70 °C|2 min|
|Cool|20 °C|5 min|

* Add 25 μl of KAPA Bead Binding Buffer to the mixture of KAPA mRNA Capture Beads and RNA and mix thoroughly by gently pipetting up and down several times.
* Incubate the plate/tube at 20 °C for 5 min, if you have a shaker that is preferable.
* Place the plate/tube on the magnet and incubate at room temperature until the solution is clear. Remove and discard the supernatant.
* Remove the beads from the magnet and resuspend in 100 μl of KAPA mRNA Bead Wash Buffer by pipetting up and down several times.
* Place the plate/tube on the magnet and incubate at room temperature until the solution is clear. Remove and discard the entire volume of supernatant.

### mRNA Elution, Fragmentation, and Priming
* Prepare the required volume of **1X** Fragment, Prime and Elute Buffer as follows:

|Component|Total Volume Needed for 4 RXNs (Includes 10% excess)|
|---------|------|
|Water| 24.2 μl|
|Fragment, Prime and Elute Buffer (2X)| 24.2μl|
|**Total Mix Volume**| **48.4 μl**|
|**Ammount added to each tube**|**11 μl**|

* Thoroughly resuspend the KAPA mRNA Capture Beads with captured mRNA in 11 μl of 1X Fragment, Prime and Elute Buffer.

---

### Safe Stopping Point
Resuspended beads with captured mRNA may be stored at 4 oC for up to 24 hours. Do not freeze the samples as this will damage the beads. When ready, proceed to step below.

---

* Place the plate/tubes in a thermal cycler and carry out the fragmentation and priming program as follows:

|Desired Fragment Size| Temp.| Duration|
|---------------------|------|---------|
|100 – 200 bp|94 °C|8 min|
|200 – 300 bp|94 °C|6 min|
|300 – 400 bp|85 °C|6 min|

##### For Puritz and Lotterhos (2017), we chose 94 °C for 7 mins to have fragments between 150-250 bp, approximately the same size distribution as planned for our DNA libraries.

* Immediately place the plate/tube on a magnet to capture the beads, and incubate until the liquid is clear.

**Caution: To prevent hybridization of poly(A)- rich RNA to the capture beads, do not allow the sample to cool before placing on the magnet.**
* Carefully remove 10 μl of the supernatant containing the eluted, fragmented, and primed RNA into a separate plate or tube.
* Proceed immediately to **1st Strand Synthesis**.

### 1st Strand Synthesis
* On ice, assemble the 1st Strand Synthesis reaction as follows:

|Component|Volume|
|---------|------|
|Fragmented, primed RNA eluted from beads| 10 μl|
|1st Strand Synthesis Master Mix| 5 μl|
|**Total Volume**| **15 μl**|

* Keeping the plate/tube on ice, mix thoroughly by gently pipetting the reaction up and down several times.
* Incubate the plate/tube using the following protocol:

|Step|Temp.|Duration|
|----|-----|--------|
|Primer extension|25 °C|10 min|
|1st Strand synthesis|42 °C|15 min|
|Enzyme inactivation|70 °C|15 min|
|HOLD|4 °C|∞|

* Place the plate/tube on ice and proceed immediately to **2nd Strand Synthesis and Marking**.

### 2nd Strand Synthesis and Marking
* Assemble the 2nd Strand Synthesis and Marking reaction as follows:

|Component|Volume|
|---------|------|
|1st Strand cDNA| 15 μl|
|2nd Strand Synthesis and Marking Master Mix| 15 μl|
|**Total Volume**| **30 μl**|

* Mix thoroughly by gently pipetting the reaction up and down several times.
* Incubate the plate/tube using the following protocol:

|Step|Temp.|Duration|
|----|-----|--------|
|2nd Strand synthesis and marking|16 °C|60 min|
|HOLD|4 °C|∞|

* Place the plate/tube on ice and proceed immediately to **2nd Strand Synthesis and Marking Cleanup**.

### Cleanup

* Perform a 1.8X SPRI® cleanup by combining the following:

|Component|Volume|
|---------|------|
|2nd Strand Synthesis reaction product| 30 μl|
|Agencourt® AMPure® XP reagent| 54 μl|
|**Total Volume**| **84 μl**|

* Thoroughly resuspend the beads by pipetting up and down multiple times.
* Incubate the plate/tube at room temperature for 15 min to allow the DNA to bind to the beads, if you can have it shaking that is ideal.
* Place the plate/tube on a magnet to capture the beads. Incubate until the liquid is clear.
* Carefully remove and discard 74 μl of supernatant.
* Keeping the plate/tube on the magnet, add 200 μl of 80% ethanol.
* Incubate the plate/tube at room temperature for ≥30 sec.
* Carefully remove and discard the ethanol.
* Keeping the plate/tube on the magnet, add 200 μl of 80% ethanol.
* Incubate the plate/tube at room temperature for ≥30 sec.
* Carefully remove and discard the ethanol. Try to remove all residual ethanol without disturbing the beads.
* Dry the beads at room temperature, until all of the ethanol has evaporated.
  * **Caution: over-drying the beads may result in dramatic yield loss.**
* Proceed immediately to **A-Tailing** immediately, or follow the Safe Stopping Point instructions below.

---

### SAFE STOPPING POINT
* Resuspend the beads in 7.5 μl 1X A-Tailing Buffer (see table above), cover the reaction and store at 4 oC for up to 24 hours. Do not freeze the samples as this will damage the AMPure® XP® beads. When ready, proceed to **A-Tailing after Safe Stopping Point**.

---

### A-Tailing
* A-Tailing is performed either directly after the 2nd Strand Synthesis and Marking Cleanup, or after the Safe Stopping Point, where beads were resuspended in 1X A-Tailing Buffer and stored at 4 °C for up to 24 hours.

#### A-Tailing immediately
* Assemble the A-Tailing reaction as follows:

|Component|Volume|
|---------|------|
|Beads with dscDNA| --|
|A-Tailing Master Mix| 15 μl|
|**Total Volume per Tube**| **15 μl**|

* Mix thoroughly by pipetting up and down several times.
* Incubate the plate/tube using the following protocol:

|Step|Temp.|Duration|
|----|-----|--------|
|A-Tailing|30 °C|30 min|
|Enzyme inactivation|60 °C|30 min|
|HOLD|4 °C|∞|

* Proceed immediately to **Adapter Ligation**.

#### A-Tailing after safe stopping point
* To resume library preparation, combine the following reagents to perform A-Tailing:

|Component|Volume|
|---------|------|
|Beads with dscDNA (in 1X A-Tailing Buffer)| 7.5 μl |
|A-Tailing Master Mix after Safe Stopping Point| 7.5 μl|
|**Total Volume per Tube**| **15 μl**|

* Mix thoroughly by pipetting up and down several times.
* Incubate the plate/tube using the following protocol:

|Step|Temp.|Duration|
|----|-----|--------|
|A-Tailing|30 °C|30 min|
|Enzyme inactivation|60 °C|30 min|
|HOLD|4 °C|∞|

* Proceed immediately to **Adapter Ligation**.

### Adapter Ligation

#### Adapter concentration will vary depending on overall RNA yield, see table below:

|Quantity of starting material|Adapter stock concentration|Adapter concentration in ligation reaction|
|----|----|-----|
|100 – 250 ng|140 nM| 10 nM|
|251 – 500 ng|350 nM|25 nM|
|501 – 2000 ng|700 nM|50 nM|
|2001 – 4000 ng|1400 nM|100 nM|

For Puritz and Lotterhos 2017, we used 4000 ng starting RNA, but because of difficulties assessing and quantifying molluscan RNA, we chose to use a 700 nM working stock with a final reaction concentration of 50 nM.

#### This will be where we insert the custom adapters that are barcoded with RE sites

* Set up the adapter ligation reactions as follows:

|Component|Volume|
|---------|------|
|Beads with A-tailed DNA| 15 μl |
|Adapter Ligation Master Mix| 17.5 μl |
|**Adapters***| 2.5 μl|
|**Total Volume per Tube**| **35 μl**|


* Mix thoroughly by pipetting up and down several times to resuspend the beads.
* Incubate the plate/tube at 20 °C for 30 min.
* Proceed immediately to **1st Post-Ligation Cleanup**.

### Post-Ligation Cleanup

* Perform a 1X SPRI® cleanup by combining the following:

|Component|Volume|
|---------|------|
|Beads with adapter-ligated DNA| 35 μl|
|Agencourt® AMPure® XP reagent| 35 μl|
|**Total Volume per Tube**| **70 μl**|

* Thoroughly resuspend the beads by pipetting up and down multiple times.
* Incubate the plate/tube at room temperature for 15 min to allow the DNA to bind to the beads, if you can have it shaking that is ideal.
* Place the plate/tube on a magnet to capture the beads. Incubate until the liquid is clear.
* Carefully remove and discard 65 μl of supernatant.
* Keeping the plate/tube on the magnet, add 200 μl of 80% ethanol.
* Incubate the plate/tube at room temperature for ≥30 sec.
* Carefully remove and discard the ethanol.
* Keeping the plate/tube on the magnet, add 200 μl of 80% ethanol.
* Incubate the plate/tube at room temperature for ≥30 sec.
* Carefully remove and discard the ethanol. Try to remove all residual ethanol without disturbing the beads.
* Dry the beads at room temperature, until all of the ethanol has evaporated. **Caution: over-drying the beads may result in dramatic yield loss.**
* Remove the plate/tube from the magnet.
* Thoroughly resuspend the beads in 25 μl of 10 mM Tris-HCl (pH 8.0).
* Incubate the plate/tube at room temperature for 5 min to allow the DNA to elute off the beads

---

### Safe Stopping Point
The solution with resuspended beads can be stored at 4 °C for up to 24 hours. Do not freeze the beads, as this can result in dramatic loss of DNA. When ready, proceed to **2nd Post-Ligation Cleanup**.

---

### 2nd Post-Ligation Cleanup

* Perform a 1X SPRI® cleanup by combining the following:

|Component|Volume|
|---------|------|
|Beads with purifed, adapter-ligated DNA| 25 μl|
|Agencourt® AMPure® XP reagent| 25 μl|
|**Total Volume per Tube**| **50 μl**|

* Thoroughly resuspend the beads by pipetting up and down multiple times.
* Incubate the plate/tube at room temperature for 15 min to allow the DNA to bind to the beads, if you can have it shaking that is ideal.
* Place the plate/tube on a magnet to capture the beads. Incubate until the liquid is clear.
* Carefully remove and discard 45 μl of supernatant.
* Keeping the plate/tube on the magnet, add 200 μl of 80% ethanol.
* Incubate the plate/tube at room temperature for ≥30 sec.
* Carefully remove and discard the ethanol.
* Keeping the plate/tube on the magnet, add 200 μl of 80% ethanol.
* Incubate the plate/tube at room temperature for ≥30 sec.
* Carefully remove and discard the ethanol. Try to remove all residual ethanol without disturbing the beads.
* Dry the beads at room temperature, until all of the ethanol has evaporated. **Caution: over-drying the beads may result in dramatic yield loss.**
* Remove the plate/tube from the magnet.
* Thoroughly resuspend the beads in 11.25 μl of 10 mM Tris-HCl (pH 8.0).
* Incubate the plate/tube at room temperature for 5 min to allow the DNA to elute off the beads.
* Place the plate/tube on a magnet to capture the beads. Incubate until the liquid is clear.
* Transfer 20 μl of the clear supernatant to a new plate/tube and proceed to *Library Amplication*.
---

### SAFE STOPPING POINT
The purified, adapter-ligated library DNA may be stored at 4 °C for up to 1 week, or frozen at -20 °C for up to 1 month. When ready, proceed to **Library Amplification**.

---
### Library Amplificiation

* Assemble each library ampli cation reaction as follows:

|Component|Volume|
|---------|------|
|Purified, adapter-ligated DNA| 10 μl|
|Library Amplification Master Mix| 15 μl|
|**Total Volume per Tube**| **25 μl**|

* Mix well by pipetting up and down several times
* Amplify the library using the following thermal cycling protocol:

|Step|Temp|Duration|Cycles|
|----|----|--------|------|
|Initial denaturation|98 °C|45 sec|1|
|Denaturation|98 °C|15 sec|12|
|Annealing*|60 °C|30 sec|12|
|Extension|72 °C|30 sec|12|
|Final Extension|72 °C|5 min|1|
|Hold|10 °C | ∞|1|


* Place the plate/tube on ice and proceed to **Library Amplification Cleanup**

### Library Amplification Cleanup

* Perform a 1X SPRI® cleanup by combining the following

|Component|Volume|
|---------|------|
|Amplified library DNA| 25 μl|
|Agencourt® AMPure® XP reagent| 25 μl|
|**Total Volume**| **50 μl**|

* Mix thoroughly by pipetting up and down several times.
* Incubate the plate/tube at room temperature for 15 min to allow the DNA to bind to the beads, if you can have it shaking that is ideal.
* Place the plate/tube on a magnet to capture the beads. Incubate until the liquid is clear.
* Carefully remove and discard 45 μl of supernatant.
* Keeping the plate/tube on the magnet, add 200 μl of 80% ethanol.
* Incubate the plate/tube at room temperature for ≥30 sec.
* Carefully remove and discard the ethanol.
* Keeping the plate/tube on the magnet, add 200 μl of 80% ethanol.
* Incubate the plate/tube at room temperature for ≥30 sec.
* Carefully remove and discard the ethanol. Try to remove all residual ethanol without disturbing the beads.
* Dry the beads at room temperature, until all of the ethanol has evaporated. **Caution: over-drying the beads may result in dramatic yield loss.**
* Remove the plate/tube from the magnet.
* Thoroughly resuspend the dried beads in 22 μl of 10 mM Tris-HCl (pH 8.0).
* Incubate the plate/tube at room temperature for 5 min to allow the DNA to elute off the beads.
* Place the plate/tube on a magnet to capture the beads. Incubate until the liquid is clear.
Transfer 21 μl of the clear supernatant to a new plate/tube.

## Quant libraries
**Procedure (Standard HS DNA protocol)**
* Set up the required number of 0.5-mL tubes for standards and samples. The Qubit® RNA HS Assay requires 2 standards.
* Label the tube lids.
* Prepare the Qubit® working solution by diluting the Qubit® DNA HS Reagent 1:200 in Qubit® DNA HS Buffer. Use a clean plastic tube each time you prepare Qubit® working solution. **Do not mix the working solution in a glass container.**
* Add 190 μL of Qubit® working solution to each of the tubes used for standards.
* Add 10 μL of each Qubit® standard to the appropriate tube, then mix by vortexing 2–3 seconds. Be careful not to create bubbles.
* Add Qubit® working solution to individual assay tubes so that the final volume in each tube after adding sample is 200 μL.
* Add each sample to the assay tubes containing the correct volume of Qubit® working solution, then mix by vortexing 2–3 seconds. The final volume in each tube should be 200 μL.
* Allow all tubes to incubate at room temperature for 2 minutes.
* On the Home screen of the Qubit® 3.0 Fluorometer, press DNA, then select DNA: High Sensitivity as the assay type. The “Read standards” screen is displayed. Press Read Standards to proceed.
* Insert the tube containing Standard #1 into the sample chamber, close the lid, then press Read standard. When the reading is complete (~3 seconds), remove Standard #1.
* Insert the tube containing Standard #2 into the sample chamber, close the lid, then press Read standard. When the reading is complete, remove Standard #2.
* Press Run samples.
* On the assay screen, select the sample volume and units
* Insert a sample tube into the sample chamber, close the lid, then press Read tube. When the reading is complete (~3 seconds), remove the sample tube.
* Repeat step last step until all samples have been read

---

### Safe Stopping Point
This is a safe stopping point. If you are stopping, store your sample at ‐15° to ‐25°C.

---


### DSN Normalization

DSN normalization is critical ensuring an even distribution of coverage across probes.  There are a genes that are highly expressed in all cells and DSN normalization helps to remove these high abundance probes and transcripts.  

#### DSN needs to be properly dilued and should be tested for activity levels before proceeding

#### The protocol below was taken from Illumina's recommendations [LINK](http://nextgen.mgh.harvard.edu/attachments/DSN_Normalization_SamplePrep_Guide_15014673_B.pdf)
#### Reagents

| Reagent| Supplier|
|----------|--------------|
|1 M HEPES buffer solution|Invitrogen, part # 15630‐080 |
|5 M NaCl solution|Ambion, part # AM9760G|
|KAPA HiFi HotStart PCR kit with dNTPs|Kapa, part #KK2502|
|Strip tubes|General lab supplier|
|DSN Kit|Evrogen, part # EA001|
|Ethanol 200 proof (absolute) for molecular biology (500 ml)|AB, part # 4333764F|
|PCR Primer PE 1.0|Included in Kapa stranded mRNA kit|
|PCR Primer PE 2.0|Included in Kapa stranded mRNA kit|
|SPRI beads|Agencourt AMPure, part # 29152; KAPA Pure Beads, part #KK8000|
|Nuclease-free water|General lab supplier|

#### Equipment
* Thermocycler (if you have two that is ideal)
* Magentic stand compatible with strip tubes

#### Procedure

### First pool individual RNA libraries in equal quantities to create a single pool, we have experiemented with 500 ng total or 200ng of each library.

* Create a 4X hybridization solution

|Component|Volume|
|---------|------|
|1 M HEPES buffer solution| 200 μl|
|5 M NaCl solution| 400 μl|
|Nuclease‐free water| 400 μl|
|**Total Volume**|**1000 μl**|

* Use two thermocyclers and set one to hold at 68°C
* Prepare the following reaction mix in a separate, sterile, nuclease‐free 200 μl PCR tube on ice for each sample to be normalized.

|Component|Volume|
|---------|------|
|Sample library| 13.5 μl|
|4X Hybridization buffer| 4.5 μl|
|**Total Volume Per Sample**|**18 μl**|

* Gently pipette the entire volume up and down 10 times, then centrifuge briefly to mix.
* Transfer the entire volume of reaction mix directly to the bottom of a new, sterile, nuclease‐free 200 μl PCR tube, using a pipette. Do not let the sample contact the side of the tube during the process.
* Incubate the reaction mix tube on the thermal cycler using the following PCR cycling conditions:

|Step|Temp|Duration|
|----|----|--------|
|Initial denaturation|98 °C|2 min|
|Treatment|68 °C|5 hours|

* **Caution**- Following incubation, keep the thermal cycler lid closed and the temperature held at 68°C. Do not remove the reaction mix tube from thermal cycler prior to and during DSN treatment.
* Dilute the 10X DSN Master buffer supplied in the DSN kit to 2X with nuclease‐ free water, 4 μl 10X DSN Master Buffer in 16 μl nuclease-free water.
* Pre‐heat the 2X DSN buffer on the second thermocycler or heat block at 68°C.
  * **Note:** Do not remove the 2X DSN buffer from the heat block during DSN treatment.
* Quickly add 20 μl of pre‐heated 2X DSN buffer to the first reaction mix tube.
* With the reaction mix tube remaining within the thermal cycler, gently pipette the entire volume up and down 10 times to mix thoroughly using a pipette set to 40 μl.
  * **Note**:Pipette the solution directly to the bottom of the PCR tube and do not let the sample contact the side of the tube during the process.
  * **Note**: It is important to keep the thermal cycler closed, except for the time necessary to add the 2X DSN buffer and mix. When preparing more than one reaction mix tube, keep the thermal cycler lid closed when extracting the 2X DSN buffer from its tube, then open the thermal cycler lid only for the time necessary to add and mix the 2X DSN buffer.
* Repeat steps 2 and 3 for each reaction mix tube.
* Incubate the reaction mix tubes on the thermal cycler at 68°C for 10 minutes.
* Quickly add 2 μl of DSN enzyme to the first reaction mix tube using a 2 μl pipette.
* With the reaction mix tube remaining within the thermal cycler, gently pipette the entire volume up and down 10 times to mix thoroughly using a pipette set to 40 μl.
  * **Note**:Pipette the solution directly to the bottom of the PCR tube and do not let the sample contact the side of the tube during the process.
* Repeat steps 6 and 7 for each reaction mix tube.
* Incubate the reaction mix tubes on the thermal cycler at 68°C for 25 minutes.
* Add 40 μl of 2X DSN stop solution to each reaction mix tube. Gently pipette the entire volume up and down to mix thoroughly, then place the tubes on ice.

---

### Safe Stopping Point
This is a safe stopping point. If you are stopping, store your sample at ‐15° to ‐25°C.

---
### SPRI Cleanup

* Perform a 1.6X SPRI® cleanup by combining the following:

|Component|Volume|
|---------|------|
|DSN Treated Library| 80 μl|
|Agencourt® AMPure® XP reagent| 128 μl|
|**Total Volume**| **208 μl**|

* Thoroughly resuspend the beads by pipetting up and down multiple times.
* Incubate the plate/tube at room temperature for 15 min to allow the DNA to bind to the beads, if you can have it shaking that is ideal.
* Place the plate/tube on a magnet to capture the beads. Incubate until the liquid is clear.
* Carefully remove and discard 200 μl of supernatant.
* Keeping the plate/tube on the magnet, add 200 μl of 80% ethanol.
* Incubate the plate/tube at room temperature for ≥30 sec.
* Carefully remove and discard the ethanol.
* Keeping the plate/tube on the magnet, add 200 μl of 80% ethanol.
* Incubate the plate/tube at room temperature for ≥30 sec.
* Carefully remove and discard the ethanol. Try to remove all residual ethanol without disturbing the beads.
* Dry the beads at room temperature, until all of the ethanol has evaporated. **Caution: over-drying the beads may result in dramatic yield loss.**
* Remove the plate/tube from the magnet.
* Thoroughly resuspend the beads in 25 μl of 10 mM Tris-HCl (pH 8.0).
* Incubate the plate/tube at room temperature for 5 min to allow the DNA to elute off the beads.
* Place the plate/tube on a magnet to capture the beads. Incubate until the liquid is clear.
* Transfer 24 μl of the clear supernatant to a new plate/tube and proceed to next step.

### PCR Enrichment of DSN Normalized Library

|Component|Volume|
|---------|------|
|DSN Treated Library | 20 μl|
|2X KAPA HiFi HotStart ReadyMix| 25 μl|
|10X KAPA Library Amplification Primer Mix| 2.5 μl|
|Nuclease-free water| 2.5 μl|
|**Total Volume per sample**| **50 μl**|

* Mix well by pipetting up and down several times
* Amplify the library using the following thermal cycling protocol:

|Step|Temp|Duration|Cycles|
|----|----|--------|------|
|Initial denaturation|98 °C|45 sec|1|
|Denaturation|98 °C|15 sec|12-14|
|Annealing*|60 °C|30 sec|12-14|
|Extension|72 °C|30 sec|12-14|
|Final Extension|72 °C|5 min|1|
|Hold|10 °C | ∞|1|

_Originally the number of cycles was set to 12, however after a second try 14 worked better for us._

### SPRI Cleanup of DSN Normalized Library

* Perform a 1.6X SPRI® cleanup by combining the following:

|Component|Volume|
|---------|------|
|Enriched DSN Library| 50 μl|
|Agencourt® AMPure® XP reagent| 80 μl|
|**Total Volume**| **130 μl**|

* Thoroughly resuspend the beads by pipetting up and down multiple times.
* Incubate the plate/tube at room temperature for 15 min to allow the DNA to bind to the beads, if you can have it shaking that is ideal.
* Place the plate/tube on a magnet to capture the beads. Incubate until the liquid is clear.
* Carefully remove and discard 115 μl of supernatant.
* Keeping the plate/tube on the magnet, add 200 μl of 80% ethanol.
* Incubate the plate/tube at room temperature for ≥30 sec.
* Carefully remove and discard the ethanol.
* Keeping the plate/tube on the magnet, add 200 μl of 80% ethanol.
* Incubate the plate/tube at room temperature for ≥30 sec.
* Carefully remove and discard the ethanol. Try to remove all residual ethanol without disturbing the beads.
* Dry the beads at room temperature, until all of the ethanol has evaporated. **Caution: over-drying the beads may result in dramatic yield loss.**
* Remove the plate/tube from the magnet.
* Thoroughly resuspend the beads in 22 μl of 10 mM Tris-HCl (pH 8.0).
* Incubate the plate/tube at room temperature for 5 min to allow the DNA to elute off the beads.
* Place the plate/tube on a magnet to capture the beads. Incubate until the liquid is clear.
* Transfer 22 μl of the clear supernatant to a new plate/tube and proceed to next step.

## Quant libraries
**Procedure (Standard HS DNA protocol)**
* Set up the required number of 0.5-mL tubes for standards and samples. The Qubit® RNA HS Assay requires 2 standards.
* Label the tube lids.
* Prepare the Qubit® working solution by diluting the Qubit® DNA HS Reagent 1:200 in Qubit® DNA HS Buffer. Use a clean plastic tube each time you prepare Qubit® working solution. **Do not mix the working solution in a glass container.**
* Add 190 μL of Qubit® working solution to each of the tubes used for standards.
* Add 10 μL of each Qubit® standard to the appropriate tube, then mix by vortexing 2–3 seconds. Be careful not to create bubbles.
* Add Qubit® working solution to individual assay tubes so that the final volume in each tube after adding sample is 200 μL.
* Add each sample to the assay tubes containing the correct volume of Qubit® working solution, then mix by vortexing 2–3 seconds. The final volume in each tube should be 200 μL.
* Allow all tubes to incubate at room temperature for 2 minutes.
* On the Home screen of the Qubit® 3.0 Fluorometer, press DNA, then select DNA: High Sensitivity as the assay type. The “Read standards” screen is displayed. Press Read Standards to proceed.
* Insert the tube containing Standard #1 into the sample chamber, close the lid, then press Read standard. When the reading is complete (~3 seconds), remove Standard #1.
* Insert the tube containing Standard #2 into the sample chamber, close the lid, then press Read standard. When the reading is complete, remove Standard #2.
* Press Run samples.
* On the assay screen, select the sample volume and units
* Insert a sample tube into the sample chamber, close the lid, then press Read tube. When the reading is complete (~3 seconds), remove the sample tube.
* Repeat step last step until all samples have been read


---

## Here we split up the cDNA libraries several ways
* One tube for sequencing (we saved 1000ng)
* Four tubes for an additional 4 cycle PCR, using the same times and temperatures as the previous PCR. Each tube contained 100ng of cDNA
	* cDNA and nuclease-free water to add up to 10μL
	* 12.5μL of 2X KAPA HiFi HotStart ReadyMix
	* 2.5μL 10X KAPA Library Amplification Primer Mix
* Remaining volume saved

### After the additional 4 cycle PCR, pool the four tubes together and perform a 1.6X SPRI® cleanup

|Component|Volume|
|---------|------|
|Enriched DSN Library| 100 μl|
|Agencourt® AMPure® XP reagent| 160 μl|
|**Total Volume**| **130 μl**|

* Thoroughly resuspend the beads by pipetting up and down multiple times.
* Incubate the plate/tube at room temperature for 15 min to allow the DNA to bind to the beads, if you can have it shaking that is ideal.
* Place the plate/tube on a magnet to capture the beads. Incubate until the liquid is clear.
* Carefully remove and discard 115 μl of supernatant.
* Keeping the plate/tube on the magnet, add 200 μl of 80% ethanol.
* Incubate the plate/tube at room temperature for ≥30 sec.
* Carefully remove and discard the ethanol.
* Keeping the plate/tube on the magnet, add 200 μl of 80% ethanol.
* Incubate the plate/tube at room temperature for ≥30 sec.
* Carefully remove and discard the ethanol. Try to remove all residual ethanol without disturbing the beads.
* Dry the beads at room temperature, until all of the ethanol has evaporated. **Caution: over-drying the beads may result in dramatic yield loss.**
* Remove the plate/tube from the magnet.
* Thoroughly resuspend the beads in 22 μl of 10 mM Tris-HCl (pH 8.0).
* Incubate the plate/tube at room temperature for 5 min to allow the DNA to elute off the beads.
* Place the plate/tube on a magnet to capture the beads. Incubate until the liquid is clear.
* Transfer 22 μl of the clear supernatant to a new plate/tube and proceed to next step.

#### Note that this step is optional as sequening the probes directly is not necessary for exome capture analysis.  Alternatively, a portion of the mRNA library can be saved before DSN normalization for sequencing.  

---

### Safe Stopping Point
This is a safe stopping point. If you are stopping, store your sample at ‐15° to ‐25°C.

---

## Probe Synthesis

Review quantifications for probes.  Ideally, there should be about 500 ng of probes per capture.  If not enough of probes is obtained, the PCR product can be re-amplified.

### Remove adapters from cDNA

#### Materials needed
| Reagent                                | Producer         | Catalog #      |            
|----------------------------------------|------------------|-----------|
|Mung Bean Nuclease| NEB| M0250S|
|SalI-HF| NEB| R3138T|
|NcoI-HF| NEB| R3193S|
|Agencourt AMPure XP  |Beckman Coulter   | A63881|

### Procedure
* Setup a restriction digest using 1 μg of DSN library
	* Each DSN library should be 1 μg in 12.25 μl

_Note: we set up 4 digestion reactions each with 1 μg of DSN library_

|Component|Total Volume Needed for 4 RXNs (Includes 10% excess)|
|---------|------|
|**Restriction Digest Master Mix**|--|
| 10X Cutsmart Buffer| 17.6 μl|
|SalI-HF Enzyme (100 units)| 4.4 μl |
|NcoI-HF Enzyme (100 units)| 4.4 μl |
| Molecular Grade H20| 95.7 μl|
|**Total Volume**| **122.1 μl**|
|**Volume added to each tube**| **27.75 μl**|

* Add 27.75 μl to each tube of 12.25 μl of DSN library for a total of 40 μl.
* Incubate reactions in thermocycler at 37°C for 4 hours and 80°C for 20 minutes to inactivate the enzymes

* **Using the same tubes from the previous step** combine the following:

|Component|Volume|
|---------|------|
|**Mung Bean Nuclease Master Mix**|--|
|10X Mung Bean Nuclease buffer| 19.8 μl|
|Mung Bean Nuclease (10 units per μl)| 2.2 μl|
|**Total Volume**| **22 μl**|
|**Volume to be Added to Each Library**| **5 μl**|

* Combine 5 μl Master Mix with Restriction Digest and DSN treated library for a total of 45 μl.
* Incubate at 30°C for 30 minutes

* Perform a 1.8X SPRI cleanup by combining the following:

#### This step may be possbile to skip and proceed directly to the 1.5X SPRI Cleanup.

### It is important to check for the digested adapters.  See trace below with adapter peaks:

![alt text](/ProbesandDigested.png)

#### If your trace looks like this, perform the additional 1.5X SPRI Cleanup below

|Component|Volume|
|---------|------|
|MBN reaction| 45 μl|
|Agencourt® AMPure® XP reagent| 81 μl|
|**Total Volume**| **126 μl**|

* Thoroughly resuspend the beads by pipetting up and down multiple times.
* Incubate the plate/tube at room temperature for 15 min to allow the DNA to bind to the beads, if you can have it shaking that is ideal.
* Place the plate/tube on a magnet to capture the beads. Incubate until the liquid is clear.
* Carefully remove and discard 115 μl of supernatant.
* Keeping the plate/tube on the magnet, add 200 μl of 80% ethanol.
* Incubate the plate/tube at room temperature for ≥30 sec.
* Carefully remove and discard the ethanol.
* Keeping the plate/tube on the magnet, add 200 μl of 80% ethanol.
* Incubate the plate/tube at room temperature for ≥30 sec.
* Carefully remove and discard the ethanol. Try to remove all residual ethanol without disturbing the beads.
* Dry the beads at room temperature, until all of the ethanol has evaporated. **Caution: over-drying the beads may result in dramatic yield loss.**
* Remove the plate/tube from the magnet.
* Thoroughly resuspend the beads in 22 μl of 10 mM Tris-HCl (pH 8.0). Volume needed depends on the number of captures. Calculate 10 μl per capture plus an aliquot for checking the probes concentration using Qubit.
* Incubate the plate/tube at room temperature for 5 min to allow the DNA to elute off the beads.
* Place the plate/tube on a magnet to capture the beads. Incubate until the liquid is clear.
* Transfer 20 μl of the clear supernatant to a new plate/tube and proceed to next step.

---

### Safe Stopping Point
This is a safe stopping point. If you are stopping, store your sample at ‐15° to ‐25°C.

---

* Perform a 1.5X SPRI cleanup by combining the following:

|Component|Volume|
|---------|------|
|MBN reaction| 22 μl|
|Agencourt® AMPure® XP reagent|33 μl|
|**Total Volume**| **55 μl**|

**If you proceeded directly to the 1.5X SPRI cleanup without doing the 1.8X SPRI cleanup first combine the following:**

|Component|Volume|
|---------|------|
|MBN reaction| 45 μl|
|Agencourt® AMPure® XP reagent|67.5 μl|
|**Total Volume**| **112.5 μl**|

* Thoroughly resuspend the beads by pipetting up and down multiple times.
* Incubate the plate/tube at room temperature for 15 min to allow the DNA to bind to the beads, if you can have it shaking that is ideal.
* Place the plate/tube on a magnet to capture the beads. Incubate until the liquid is clear.
* Carefully remove and discard 115 μl of supernatant.
* Keeping the plate/tube on the magnet, add 200 μl of 80% ethanol.
* Incubate the plate/tube at room temperature for ≥30 sec.
* Carefully remove and discard the ethanol.
* Keeping the plate/tube on the magnet, add 200 μl of 80% ethanol.
* Incubate the plate/tube at room temperature for ≥30 sec.
* Carefully remove and discard the ethanol. Try to remove all residual ethanol without disturbing the beads.
* Dry the beads at room temperature, until all of the ethanol has evaporated. **Caution: over-drying the beads may result in dramatic yield loss.**
* Remove the plate/tube from the magnet.
* Thoroughly resuspend the beads in 31 μl of 10 mM Tris-HCl (pH 8.0). **However,** volume needed depends on the number of captures. Calculate 20 μl per capture plus an aliquot for checking the probes concentration using Qubit.
* Incubate the plate/tube at room temperature for 5 min to allow the DNA to elute off the beads.
* Place the plate/tube on a magnet to capture the beads. Incubate until the liquid is clear.
* Transfer 30 μl of the clear supernatant to a new plate/tube and proceed to next step.

---

### Safe Stopping Point
This is a safe stopping point. If you are stopping, store your sample at ‐15° to ‐25°C.

---

## Biotin Labeling

### Materials needed

| Reagent                                | Producer         | Catalog #      |            
|----------------------------------------|------------------|-----------|
|DecaLabel™ Biotin DNA Labeling Kit |Thermo Scientific | K0651   |                     

### Procedure

* Add the following components into 1.5 ml microcentrifuge tube (for each Biotin Labeling):

|Component|Volume for 1 Reaction|
|---------|------|
|RE and MBN treated DSN Library| 19 μl|
|Decanucleotide in 5X Reaction Buffer| 10 μl|
| Water, nuclease-free| 15 μl|
|**Total Volume**| **44 μl**|

* Vortex the tube and spin down in a microcentrifuge for 3-5 s
* Incubate the tube in the theremocycler at 98°C for 10 minutes, bring down to a 4°C hold and once finished put it on ice. Spin down quickly.
* Add the following components in the same tube:

|Component|Volume for 1 Reaction|
|---------|------|
|Biotin Labeling Mix| 5 μl|
|Klenow fragment, exo– (5 u)| 1 μl|
|**Total Volume**| **50 μl**|

* Shake the tube and spin down in a microcentrifuge for 3-5 s.
* Incubate for 12-20 hours at 37°C.

---
#### Optional:Control reaction
* Add the following components into 1.5 ml microcentrifuge tube:

|Component|Volume|
|---------|------|
|Control Template, 10 ng/μl | 25 μl|
|Decanucleotide in 5X Reaction Buffer| 10 μl|
| Water, nuclease-free| 9 μl|
|**Total Volume**| **44 μl**|

* Vortex the tube and spin down in a microcentrifuge for 3-5 s
* Incubate the tube in a boiling water bath for 5-10 min and cool it on ice. Spin down quickly.
* Add the following components in the same tube:

|Component|Volume|
|---------|------|
|Biotin Labeling Mix| 5 μl|
|Klenow fragment, exo– (5 u)| 1 μl|
|**Total Volume**| **50 μl**|

---

* After the 12-20 hour incubation, perform a 1.5X SPRI® cleanup by combining the following:
	* _If you split your probes into multiple reactions for Biotin Labeling pool them for the bead cleanup_

|Component|Volume for 1 Biotin Reaction|
|---------|------|
|Biotin reaction| 50 μl|
|Agencourt® AMPure® XP reagent| 75 μl|
|**Total Volume**| **125 μl**|

* Thoroughly resuspend the beads by pipetting up and down multiple times.
* Incubate the plate/tube at room temperature for 15 min to allow the DNA to bind to the beads, if you can have it shaking that is ideal.
* Place the plate/tube on a magnet to capture the beads. Incubate until the liquid is clear.
* Carefully remove and discard 115 μl of supernatant.
* Keeping the plate/tube on the magnet, add 200 μl of 80% ethanol.
* Incubate the plate/tube at room temperature for ≥30 sec.
* Carefully remove and discard the ethanol.
* Keeping the plate/tube on the magnet, add 200 μl of 80% ethanol.
* Incubate the plate/tube at room temperature for ≥30 sec.
* Carefully remove and discard the ethanol. Try to remove all residual ethanol without disturbing the beads.
* Dry the beads at room temperature, until all of the ethanol has evaporated. **Caution: over-drying the beads may result in dramatic yield loss.**
* Remove the plate/tube from the magnet.
* Thoroughly resuspend the beads in 20 μl of 10 mM Tris-HCl (pH 8.0). **However,** volume needed depends on the number of captures. Calculate 10 μl per capture plus an aliquot for checking the probes concentration using Qubit.
* Incubate the plate/tube at room temperature for 5 min to allow the DNA to elute off the beads.
* Place the plate/tube on a magnet to capture the beads. Incubate until the liquid is clear.
* Transfer 19 μl of the clear supernatant to a new plate/tube and proceed to next step.

**If you pooled your probes after Biotin Labeling, do _THIS_ 1.5X SPRI® cleanup**

|Component|Volume for 4 Biotin Reactions|
|---------|------|
|Biotin reaction| 200 μl|
|Agencourt® AMPure® XP reagent| 300 μl|
|**Total Volume**| **500 μl**|

* Thoroughly resuspend the beads by pipetting up and down multiple times.
* Incubate the plate/tube at room temperature for 15 min to allow the DNA to bind to the beads, if you can have it shaking that is ideal.
* Place the plate/tube on a magnet to capture the beads. Incubate until the liquid is clear.
* Carefully remove and discard 450 μl of supernatant.
* Keeping the plate/tube on the magnet, add 600 μl of 80% ethanol.
* Incubate the plate/tube at room temperature for ≥30 sec.
* Carefully remove and discard the ethanol.
* Keeping the plate/tube on the magnet, add 600 μl of 80% ethanol.
* Incubate the plate/tube at room temperature for ≥30 sec.
* Carefully remove and discard the ethanol. Try to remove all residual ethanol without disturbing the beads.
* Dry the beads at room temperature, until all of the ethanol has evaporated. **Caution: over-drying the beads may result in dramatic yield loss.**
* Remove the plate/tube from the magnet.
* Thoroughly resuspend the beads in 50 μl of 10 mM Tris-HCl (pH 8.0). **However,** volume needed depends on the number of captures. Calculate 10 μl per capture plus an aliquot for checking the probes concentration using Qubit.
* Incubate the plate/tube at room temperature for 5 min to allow the DNA to elute off the beads.
* Place the plate/tube on a magnet to capture the beads. Incubate until the liquid is clear.
* Transfer 19 μl of the clear supernatant to a new plate/tube and proceed to next step.


## Quant Probes
**Procedure (Standard HS DNA protocol)**
* Set up the required number of 0.5-mL tubes for standards and samples. The Qubit® RNA HS Assay requires 2 standards.
* Label the tube lids.
* Prepare the Qubit® working solution by diluting the Qubit® DNA HS Reagent 1:200 in Qubit® DNA HS Buffer. Use a clean plastic tube each time you prepare Qubit® working solution. **Do not mix the working solution in a glass container.**
* Add 190 μL of Qubit® working solution to each of the tubes used for standards.
* Add 10 μL of each Qubit® standard to the appropriate tube, then mix by vortexing 2–3 seconds. Be careful not to create bubbles.
* Add Qubit® working solution to individual assay tubes so that the final volume in each tube after adding sample is 200 μL.
* Add each sample to the assay tubes containing the correct volume of Qubit® working solution, then mix by vortexing 2–3 seconds. The final volume in each tube should be 200 μL.
* Allow all tubes to incubate at room temperature for 2 minutes.
* On the Home screen of the Qubit® 3.0 Fluorometer, press DNA, then select DNA: High Sensitivity as the assay type. The “Read standards” screen is displayed. Press Read Standards to proceed.
* Insert the tube containing Standard #1 into the sample chamber, close the lid, then press Read standard. When the reading is complete (~3 seconds), remove Standard #1.
* Insert the tube containing Standard #2 into the sample chamber, close the lid, then press Read standard. When the reading is complete, remove Standard #2.
* Press Run samples.
* On the assay screen, select the sample volume and units
* Insert a sample tube into the sample chamber, close the lid, then press Read tube. When the reading is complete (~3 seconds), remove the sample tube.
* Repeat step last step until all samples have been read

## Visualize Probes

* Run probes on BioAnalyzer/Tape Station/Fragment analyzer

They should look like this:

![alt text](/FinalProbes.png)

---

### Safe Stopping Point
This is a safe stopping point. If you are stopping, store your sample at ‐15° to ‐25°C. Your probes are done!

---

## Preparation of whole genome libraries

### Using KAPA HyperPrep Kit with 1/2 reactions

Refer to [manual](https://www.kapabiosystems.com/document/kapa-hyper-prep-kit-tds/?dl=1) during procedure (steps below are for notes and comments).

#### For Puritz and Lotterhos 2017, genomic DNA was sheared to a modal peak of 150 bp using a Covaris sonicator.  The protocol below assumes you are starting with sheared DNA

### Materials Needed

Adapter Oligos:

|Name| 5' to 3' Sequence|
|----------|---------------------------------------------|
|DNA_P1.1.1|	ACACTCTTTCCCTACACGACGCTCTTCCGATCTGCATGG*T|
|DNA_P1.1.2|	ACACTCTTTCCCTACACGACGCTCTTCCGATCTAACCAG*T|
|DNA_P1.1.3|	ACACTCTTTCCCTACACGACGCTCTTCCGATCTCGATCG*T|
|DNA_P1.1.4|	ACACTCTTTCCCTACACGACGCTCTTCCGATCTTCGATG*T|
|DNA_P1.1.5|	ACACTCTTTCCCTACACGACGCTCTTCCGATCTTGCATG*T|
|DNA_P1.1.6|	ACACTCTTTCCCTACACGACGCTCTTCCGATCTCAACCG*T|
|DNA_P1.1.7|	ACACTCTTTCCCTACACGACGCTCTTCCGATCTGGTTGG*T|
|DNA_P1.1.8|	ACACTCTTTCCCTACACGACGCTCTTCCGATCTAAGGAG*T|
|DNA_P1.2.1|	PC*CATGCAGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGT|
|DNA_P1.2.2|	PC*TGGTTAGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGT|
|DNA_P1.2.3|	PC*GATCGAGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGT|
|DNA_P1.2.4|	PC*ATCGAAGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGT|
|DNA_P1.2.5|	PC*ATGCAAGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGT|
|DNA_P1.2.6|	PC*GGTTGAGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGT|
|DNA_P1.2.7|	PC*CAACCAGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGT|
|DNA_P1.2.8|	PC*TCCTTAGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGT|
|DNA_P2.1|	P*GATCGGAAGAGCGAGAACAA|
|DNA_P2.2|	GTGACTGGAGTTCACACGTGTGCTCTTCCGATC*T|

PCR Primers:

|Name| 5' to 3' Sequence|
|----------|---------------------------------------------|
|PCR1_P5_primer|AATGATACGGCGACCACCGAGATCTACACTCTTTCCCTACACGACGCTCTTCCGATC*T|
|PCR2_01_ATCACG|CAAGCAGAAGACGGCATACGAGATCGTGATGTGACTGGAGTTCAGACGTGTG*C|
|PCR2_02_CGATGT|CAAGCAGAAGACGGCATACGAGATACATCGGTGACTGGAGTTCAGACGTGTG*C|
|PCR2_03_TTAGGC|CAAGCAGAAGACGGCATACGAGATGCCTAAGTGACTGGAGTTCAGACGTGTG*C|
|PCR2_04_TGACCA|CAAGCAGAAGACGGCATACGAGATTGGTCAGTGACTGGAGTTCAGACGTGTG*C|
|PCR2_05_ACAGTG|CAAGCAGAAGACGGCATACGAGATCACTGTGTGACTGGAGTTCAGACGTGTG*C|
|PCR2_06_GCCAAT|CAAGCAGAAGACGGCATACGAGATATTGGCGTGACTGGAGTTCAGACGTGTG*C|
|PCR2_07_CAGATC|CAAGCAGAAGACGGCATACGAGATGATCTGGTGACTGGAGTTCAGACGTGTG*C|
|PCR2_08_ACTTGA|CAAGCAGAAGACGGCATACGAGATTCAAGTGTGACTGGAGTTCAGACGTGTG*C|
|PCR2_09_GATCAG|CAAGCAGAAGACGGCATACGAGATCTGATCGTGACTGGAGTTCAGACGTGTG*C|
|PCR2_10_TAGCTT|CAAGCAGAAGACGGCATACGAGATAAGCTAGTGACTGGAGTTCAGACGTGTG*C|
|PCR2_11_GGCTAC|CAAGCAGAAGACGGCATACGAGATGTAGCCGTGACTGGAGTTCAGACGTGTG*C|
|PCR2_12_CTTGTA|CAAGCAGAAGACGGCATACGAGATTACAAGGTGACTGGAGTTCAGACGTGTG*C|

### Anneal Adapters
Single-stranded oligos need to be annealed with their appropriate partner before ligation.

* To create Adapter P1, combine each oligo 1.1 with its complementary oligo 1.2 in a 1:1 ratio in working strength annealing buffer (final buffer concentration 1x) for a total annealed adapter concentration of 40uM (for example, if purchased oligos are resuspended to an initial concentration of 100uM, use 40ul oligo 1.1, 40ul oligo 1.2, 10ul 10x annealing buffer and 10ul nuclease-free water). Do the same for oligos 2.1 and 2.2 to create the common adapter P2.
* In a thermocyler, incubate at 97.5°C for 2.5 minutes, and then cool at a rate of not greater than 3°C per minute until the solution reaches a temperature of 21°C. Hold at 4°C.
* Prepare final working strength concentrations of annealed adapters from this annealed stock. For convenience, it is possible to store the adapters at 4°C while in active use.  **For Puritz and Lotterhos 2017, a working stock of 40 μM was used, leading to a final adapter:insert molar ratio of ~ 50:1.**

### Note about multiplexing and DNA adapters

The above adapters have inline barcodes which will appear in the sequences and need to be demultiplexed bioinformatically.

|Adapter| Inline Barcode|
|-------|---------------|
|1|GCATG|
|2|AACCA|
|3|CGATC|
|4|TCGAT|
|5|TGCAT|
|6|CAACC|
|7|GGTTG|
|8|AAGGA|

These inline barcodes can be combined with the 12 PCR primers that add a P7 Illumina index which is read directly in a sepeartate sequencing run and is returned to the user in index-labeled files.  With the above design 96 different individuals or pools could be multiplexed on a single lane.

Alternatively, any TruSeq style DNA adapters could be used with this protocol as long as they match with the [Blockling Oligos](#blocking-oligos) used during [Probe Hybridization](#hybridization)
---

### Procedure

### End repair
* Adjust sample volume of 500 ng fragmented DNA to 25 μl.
* Add the following to each sample:

|Component|Total Volume Needed for 4 RXNs (Includes 10% excess)|
|---------|------|
|**End Repair and A-Tailing Master Mix:**|--|
|End Repair & A-Tailing Buffer | 15.4 μl|
|End Repair & A-Tailing Enzyme Mix | 6.6 μl|
|**Total Volume**| **22 μl**|
|**Volume to add to each fragmented/sheared DNA sample**| **5 μl**|

  * The buffer and enzyme mix should preferably be pre-mixed and added in a single pipetting step.
    * Premixes are stable for ≤24 hrs at room temperature, for ≤3 days at 4°C, and for ≤4 weeks at -20°C
* Add 5 μl of End Repair and A-Tailing Master Mix to each 25 μl of fragmented/sheared DNA for a total of 30 μl.
* Vortex gently and spin down briefly. Return the reaction plate/tube(s) to ice. Proceed immediately to the next step.
* Incubate in a thermocycler programmed as outlined below.

|Step|Temp|Time|
|----|----|--------|
|End repair and A-tailing 1|20 °C|30 min|
|End repair and A-tailing 2|65 °C|30 min|
|Hold|10 °C | ∞|

* **Notes**
  * A heated lid is required for this incubation. If possible, set the temperature of the lid at 85°C, instead of the usual ~105°C.
  * If proceeding to the adapter ligation reaction setup without any delay, the reaction may be cooled to 20°C instead of 4°C.

### Adapter ligation
* Dilute adapter stocks to the appropriate concentration, as outlined below:

|Fragmented DNA| Adapter stock concentration | Adapter:insert molar ratio|
|--------------|-----------------------------|---------------------------|
| 1 μg|15 μM|10:1|
|500 ng|15 μM|20:1|
|250 ng|15 μM|40:1|
|100 ng|15 μM|100:1|
|50 ng|15 μM|200:1|
|25 ng|15 μM|200:1|
|10 ng|15 μM|200:1|
|5 ng|15 μM|200:1|
|2.5 ng|15 μM|200:1|
|1 ng|15 μM|200:1|

**For Puritz and Lotterhos (2017), a working stock of 40 μM was used, leading to a final adapter:insert molar ratio of ~ 50:1.**

* In the same plate/tube(s) in which end repair and A-tailing was performed, assemble each adapter ligation reaction as follows:

|Component|Total Volume Needed for 4 RXNs (Includes 10% excess)|
|---------|------|
|**Ligation Master Mix**|--|
|Ligation Buffer| 66 μl|
|DNA Ligase| 22 μl|
|PCR Grade Water| 11 μl|
|**Total Volume**| **99 μl**|
|**Volume added to Each Tube**| **22.5 μl**|

* Then to the tube with the aliqiots of Ligation Master Mix add:

|Component| Volume for 1 Sample|
|---------|------|
|End repair and A-tailing reaction product| 30 μl|
|P1 Adapter stock (concentration as required) **Barcode containing oligo** | 1.25 μl|
|P2 Adapter stock (concentration as required) | 1.25 μl|
|**Total Volume**| **55 μl**|

* Notes
  * **Each P1 adapter has a unique barcode** these barcodes can be combined with Illumina indices (added later via PCR) for high levels of multiplexing.
  * The water, buffer and ligase enzyme should preferably be premixed and added in a single pipetting step. Premixes are stable for ≤24 hrs at room temperature, for ≤3 days at 4°C, and for ≤4 weeks at -20°C.

* Mix thoroughly and centrifuge briefly.
* Incubate at 20°C for 60 min, if you can have it shaking that is ideal.
  * Note: to achieve higher conversion rates and library yields, particularly for low-input samples, consider increasing the ligation time—to a maximum of 4 hrs at 20°C, or overnight at 4°C. Please note that longer ligation times may lead to increased levels of adapter-dimer. Adapter concentrations may have to be optimized if ligation times are extended signifcantly.
* Proceed immediately to the next step.

### Post-ligation Cleanup
* In the same plate/tube(s), perform a 0.8X bead- based cleanup by combining the following:

|Component|Volume|
|---------|------|
|Adapter ligation reaction product| 55 μl|
|KAPA Pure Beads | 44 μl|
|**Total Volume**| **99 μl**|

* Mix thoroughly by vortexing and/or pipetting up and down multiple times.
* Incubate the plate/tube(s) at room temperature for 15 min to bind DNA to the beads, if you can have it shaking that is ideal.
* Place the plate/tube(s) on a magnet to capture the beads. Incubate until the liquid is clear.
* Carefully remove and discard the supernatant.
* Keeping the plate/tube(s) on the magnet, add 200 μL of 80% ethanol.
* Incubate the plate/tube(s) on the magnet at room temperature for ≥30 sec.
* Carefully remove and discard the ethanol.
* Keeping the plate/tube(s) on the magnet, add 200 μL of 80% ethanol.
* Incubate the plate/tube(s) on the magnet at room temperature for ≥30 sec.
* Carefully remove and discard the ethanol. Try to remove all residual ethanol without disturbing the beads.
* Dry the beads at room temperature for 3 – 5 min, or until all of the ethanol has evaporated. *Caution: over-drying the beads may result in reduced yield.*
* Remove the plate/tube(s) from the magnet.
* Thoroughly resuspend the beads in in 12.5 μL of elution buffer (10 mM Tris-HCl, pH 8.0 – 8.5)
* Incubate the plate/tube(s) at room temperature for 5 min to elute DNA off the beads.
* Place the plate/tube(s) on a magnet to capture the beads. Incubate until the liquid is clear.
* Transfer 11 μL of supernatant to a new plate/tube(s):

### Quant samples
**Procedure (Standard HS DNA protocol)**
* Set up the required number of 0.5-mL tubes for standards and samples. The Qubit® RNA HS Assay requires 2 standards.
* Label the tube lids.
* Prepare the Qubit® working solution by diluting the Qubit® DNA HS Reagent 1:200 in Qubit® DNA HS Buffer. Use a clean plastic tube each time you prepare Qubit® working solution. **Do not mix the working solution in a glass container.**
* Add 190 μL of Qubit® working solution to each of the tubes used for standards.
* Add 10 μL of each Qubit® standard to the appropriate tube, then mix by vortexing 2–3 seconds. Be careful not to create bubbles.
* Add Qubit® working solution to individual assay tubes so that the final volume in each tube after adding sample is 200 μL.
* Add each sample to the assay tubes containing the correct volume of Qubit® working solution, then mix by vortexing 2–3 seconds. The final volume in each tube should be 200 μL.
* Allow all tubes to incubate at room temperature for 2 minutes.
* On the Home screen of the Qubit® 3.0 Fluorometer, press DNA, then select DNA: High Sensitivity as the assay type. The “Read standards” screen is displayed. Press Read Standards to proceed.
* Insert the tube containing Standard #1 into the sample chamber, close the lid, then press Read standard. When the reading is complete (~3 seconds), remove Standard #1.
* Insert the tube containing Standard #2 into the sample chamber, close the lid, then press Read standard. When the reading is complete, remove Standard #2.
* Press Run samples.
* On the assay screen, select the sample volume and units
* Insert a sample tube into the sample chamber, close the lid, then press Read tube. When the reading is complete (~3 seconds), remove the sample tube.
* Repeat step last step until all samples have been read

### Pool samples to be used with in the same index/capture.  

Each P1 adapter has a unique barcode sequence. These barcodes can be combined with Illumina indices (added later via PCR) for high levels of multiplexing within in a single lane of Illumina sequencing.  At this point samples with different barcodes but the same Illumina index can be pooled before amplification.  The current version of the protocol supports pooling 12 individuals per Illumina index, along with 12 unique Illumina Indices.

### Library Amplification

* Assemble each library amplification reaction as follows per pool:

|Component|Volume|
|---------|------|
|KAPA HiFi HotStart ReadyMix (2X) | 12.5 μl|
|PCR1_P5 Primer (Universal)  | 1.25 μl|
|PCR2 Primer (Indexed)  | 1.25 μl|
|Adapter-ligated library pool| 10.0 μl|
|**Total Volume**| **25 μl**|

* Calculate number of cycles needed based on previous quants
* You can make a mix of your primers if they are the same for each sample for faster pipetting

|Amount of adapter-ligated DNA in amplification reaction| Number of cycles required to generate 1 μg of library DNA|
|--------------|-------------------------------------------|
|500 ng|1-2|
|100 ng|3-4|
|50 ng|5-6|
|25 ng|7-8|
|10 ng|8-9|
|5 ng|11-12|
|1 ng|12-13|

### For Puritz and Lotterhos (2017), 6 PCR cycles were used.

* Mix thoroughly and centrifuge briefly.
* Amplify using the following cycling protocol:

|Step|Temp|Duration|Cycles|
|----|----|--------|------|
|Initial denaturation|98 °C|45 sec|1|
|Denaturation|98 °C|15 sec|X|
|Annealing*|60 °C|30 sec|X|
|Extension|72 °C|30 sec|X|
|Final Extension|72 °C|1 min|1|
|Hold|4 °C | ∞|1|

* Proceed immediately to the next step

### Post-amplification Cleanup

* In the library amplification plate/tube(s) perform a 1X bead-based cleanup by combining the following:

|Component|Volume|
|---------|------|
|Adapter ligation reaction product| 25 μl|
|KAPA Pure Beads | 25 μl|
|**Total Volume**| **50 μl**|

* Mix thoroughly by vortexing and/or pipetting up and down multiple times
* Incubate the plate/tube(s) at room temperature for 15 min to bind DNA to the beads, if you can have it shaking that is ideal.
* Place the plate/tube(s) on a magnet to capture the beads. Incubate until the liquid is clear
* Carefully remove and discard the supernatant
* Keeping the plate/tube(s) on the magnet, add 200 μL of 80% ethanol.
* Incubate the plate/tube(s) on the magnet at room temperature for ≥30 sec.
* Carefully remove and discard the ethanol.
* Keeping the plate/tube(s) on the magnet, add 200 μL of 80% ethanol.
* Incubate the plate/tube(s) on the magnet at room temperature for ≥30 sec.
* Carefully remove and discard the ethanol. Try to remove all residual ethanol without disturbing the beads.
* Dry the beads at room temperature for 3 – 5 min, or until all of the ethanol has evaporated. Caution: over-drying the beads may result in reduced yield.
* Remove the plate/tube(s) from the magnet.
* Resuspend in 16 μl of 10 mM Tris or water
* Incubate the plate/tube(s) at room temperature for 5 min to elute DNA off the beads.
* Place the plate/tube(s) on a magnet to capture the beads. Incubate until the liquid is clear.
* Transfer 15 μL of supernatant to a new plate/tube(s):

---

### Safe Stopping Point
This is a safe stopping point. If you are stopping, store your sample at ‐15° to ‐25°C.

---
## Hybridization and Capture

### Materials needed
| Reagent                                | Supplier         | Catalog #      |            
|----------------------------------------|------------------|-----------|
|Denhardt’s solution  (50x)              |Life Technologies | 750018        |           
|Dynabeads® M-280 Streptavidin           |Life Technologies | 11205D, M-270 |           
|SSC Buffer Concentrate (20x)            |Fisher Scientific | 5075059      |              
|SDS Micropellets           |Fisher Scientific | BP8200100      |             
|Cot-1 DNA (1 mg/ml)                     |ThermoFischer     | 15279011|                   
|Agencourt AMPure XP                     |Beckman Coulter   | A63881|

## Blocking Oligos

**Note that if using custom DNA adapters, it is critical to ensure that the blocking oligos match the adapter sequences.**

|Name       |5' to 3' Sequence                                                |
|-----------|------------------------------------------------------------------|
| BO1.P5.F  | AATGATACGGCGACCACCGAGATCTACACTCTTTCCCTACACGACGCTCTTCCGATCT       |
| BO2.P5.R  | AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTAGATCTCGGTGGTCGCCGTATCATT       |
| BO3.P7.F  | CAAGCAGAAGACGGCATACGAGATIIIIIIGTGACTGGAGTTCAGACGTGTGCTCTTCCGATCT |
| BO4.P7.R  | AGATCGGAAGAGCACACGTCTGAACTCCAGTCACIIIIIIATCTCGTATGCCGTCTTCTGCTTG |


Solutions needed:

* 10 mM Tris-HCl pH 8.5 or PCR-grade water
* EDTA 500 mM
* SDS 10%
* TEN (10 mM Tris-HCl pH 7.5, 1 mM EDTA, 1M NaCl)
* 1x SSC / 0.1% SDS
* 0.5x SSC / 0.1% SDS
* 0.1x SSC / 0.1% SDS

Protocol based on previously described methods [hyRAD](https://github.com/chiasto/hyRAD/blob/master/wetlab.md#4-hybridization-capture-and-library-re-amplification) and [general capture](http://openwetware.org/wiki/Hyb_Seq_Prep)

Remember to perform one capture per pool of the libraries amplified with the same Illumina indexed primer.

## Hybridization

* Prepare the hybridization mix. Probes and blocking oligos are used in excess. We recommend using 500 ng of probes with 500 ng of gDNA library, but have also successfully captured with as little as 100 ng of probes and 100 ng of library.  **Yield will scale directly with input**

| Component                                | Volume for 1 Hybridization |
| -------------------------------------- | ----------- |
| Molecular Grade Water                  | 3.5 μl        |
| SSC (20x)                              | 12.0  μl      |
| EDTA (500 mM)                          | 0.4 μl        |
| SDS (10%)                              | 0.4 μl        |
| Denhardt’s solution (50x)              | 1.6 μl        |
| Cot-1 DNA (1 mg/ml)                    | 0.5 μl        |
| BO.1 blocking oligo (200 μM)           | 0.4 μl        |
| BO.2 blocking oligo (200 μM)           | 0.4 μl        |
| BO.3 blocking oligo (200 μM)           | 0.4 μl        |
| BO.4 blocking oligo (200 μM)           | 0.4 μl        |
| prepared Illumina library (X ng)| X μl        |
| probes (X ng)               | X μl         |

**If gDNA library is pooled amounts (ex. 200ng of each library or 500ng total) the volume will vary between pools. The volume of the probes will vary too. To end up with 40 μl of hybridization mixture, just make sure that the volume of the probes, gDNA and water adds up to 23.5 μl. The rest of the components can be combined into a master mix and 16.5 μl of that added to each hybridization reaction tube.**

_What have worked for us is 4 pools of gDNA, 6 samples with 200ng each._

* Incubate at 95°C for 10 minutes, then at 65°C for 24-48 hours. Mix from time to time. This can be done in a standard thermocycler, but is probably best performed in an hybridization oven with a rotor.

### Preparation of Dynabeads

* Resuspend well Dynabeads M-280 (10 mg/ml).
* Dispense 10 μl of beads in a PCR tube.
* Wash:
	* magnetize, remove and discard supernatant,
	* resuspend in 200 μl of TEN.
* Repeat the wash two times for a total of three washes
* Store in 200 μl of TEN at room temperature until use.

If more captures are expected, increase the initial amount of beads accordingly, transfer the final resupension into an eppendorf tube and add the appropriate volume of TEN (10 μl of beads should be resuspended in 200 μl of TEN).

### Washes

**The first two washes have to be warmed to 65°C**

* Add 40 μl of the hybridization mixture to 200ul of Dynabeads
* Gently mix with pippette or inverting tube
* Incubate 30 min at room temperature.
* Place on the magnet
* Remove supernatant and retain in case of DNA loss.
* Resuspend beads in 200 μl of **65°C 1x SSC / 0.1% SDS**.
* Mix well and incubate for 15 min, 65°C.
* Place on the magnet
* Remove supernatant and retain in case of DNA loss.
* Replace with 200 μl of **65°C 1x SSC / 0.1% SDS**.
* Mix well and incubate for 10 min, 65°C.
* Place on the magnet
* Remove supernatant and retain in case of DNA loss.
* Replace with 200 μl of **0.5x SSC / 0.1% SDS**.
* Mix well, incubate for 10 min, 65°C.
* Place on the magnet
* Remove supernatant and retain in case of DNA loss.
* Replace with 200 μl of **0.1x SSC / 0.1% SDS**.
* Mix well, incubate for 10 min, 65°C.
* Place on the magnet
* Remove supernatant and retain in case of DNA loss.
* Replace with 22 μl of 80°C H20
* Mix well, incubate for for 10 min, 80°C.
* Place on magnet
* Remove and **SAVE THE SUPERNATANT** 21 μl (this contains the hybridization-enriched fragments)
* Discard the beads.						

### Library re-amplification

* Assemble each library ampli cation reaction as follows:

|Component|Volume|
|---------|------|
|KAPA HiFi HotStart ReadyMix (2X) | 12.5 μl|
|KAPA Primer Mix (Universal)  | 2.5 μl|
|Enriched Library| 10.0 μl|
|**Total Volume**| **25 μl**|

**NOTE:*** It's important to use the same INDEX primer as the original library prep!

* Mix thoroughly and centrifuge briefly.
* Amplify using the following cycling protocol:

|Step|Temp|Duration|Cycles|
|----|----|--------|------|
|Initial denaturation|98 °C|45 sec|1|
|Denaturation|98 °C|15 sec|6-12|
|Annealing*|60 °C|30 sec|6-12|
|Extension|72 °C|30 sec|6-12|
|Final Extension|72 °C|1 min|1|
|Hold|4 °C | ∞|1|

_We decided on using 12 cycles because that gave the desired amplification._

* Perform a 1X SPRI® cleanup by combining the following:

|Component|Volume|
|---------|------|
|Hybridized and amplified reaction| 25 μl|
|Agencourt® AMPure® XP reagent|  25 μl|
|**Total Volume**| **50 μl**|

* Thoroughly resuspend the beads by pipetting up and down multiple times.
* Incubate the plate/tube at room temperature for 15 min to allow the DNA to bind to the beads, f you can have it shaking that is ideal.
* Place the plate/tube on a magnet to capture the beads. Incubate until the liquid is clear.
* Carefully remove and discard 115 μl of supernatant.
* Keeping the plate/tube on the magnet, add 200 μl of 80% ethanol.
* Incubate the plate/tube at room temperature for ≥30 sec.
* Carefully remove and discard the ethanol.
* Keeping the plate/tube on the magnet, add 200 μl of 80% ethanol.
* Incubate the plate/tube at room temperature for ≥30 sec.
* Carefully remove and discard the ethanol. Try to remove all residual ethanol without disturbing the beads.
* Dry the beads at room temperature, until all of the ethanol has evaporated. **Caution: over-drying the beads may result in dramatic yield loss.**
* Remove the plate/tube from the magnet.
* Thoroughly resuspend the beads in 25 μl of 10 mM Tris-HCl (pH 8.0).
* Incubate the plate/tube at room temperature for 5 min to allow the DNA to elute off the beads.
* Place the plate/tube on a magnet to capture the beads. Incubate until the liquid is clear.
* Transfer 24 μl of the clear supernatant to a new plate/tube and proceed to next step.


### Quant samples
**Procedure (Standard HS DNA protocol)**
* Set up the required number of 0.5-mL tubes for standards and samples. The Qubit® RNA HS Assay requires 2 standards.
* Label the tube lids.
* Prepare the Qubit® working solution by diluting the Qubit® DNA HS Reagent 1:200 in Qubit® DNA HS Buffer. Use a clean plastic tube each time you prepare Qubit® working solution. **Do not mix the working solution in a glass container.**
* Add 190 μL of Qubit® working solution to each of the tubes used for standards.
* Add 10 μL of each Qubit® standard to the appropriate tube, then mix by vortexing 2–3 seconds. Be careful not to create bubbles.
* Add Qubit® working solution to individual assay tubes so that the final volume in each tube after adding sample is 200 μL.
* Add each sample to the assay tubes containing the correct volume of Qubit® working solution, then mix by vortexing 2–3 seconds. The final volume in each tube should be 200 μL.
* Allow all tubes to incubate at room temperature for 2 minutes.
* On the Home screen of the Qubit® 3.0 Fluorometer, press DNA, then select DNA: High Sensitivity as the assay type. The “Read standards” screen is displayed. Press Read Standards to proceed.
* Insert the tube containing Standard #1 into the sample chamber, close the lid, then press Read standard. When the reading is complete (~3 seconds), remove Standard #1.
* Insert the tube containing Standard #2 into the sample chamber, close the lid, then press Read standard. When the reading is complete, remove Standard #2.
* Press Run samples.
* On the assay screen, select the sample volume and units
* Insert a sample tube into the sample chamber, close the lid, then press Read tube. When the reading is complete (~3 seconds), remove the sample tube.
* Repeat step last step until all samples have been read

### Verify

* Run samples on BioAnalyzer/Tape Station/Fragment analyzer
